### PR TITLE
Add more examples

### PR DESCRIPTION
**Pull request by l2yao**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

`tox -e docs`

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-experiments/blob/master/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-experiments/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-experiments/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-experiments/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/smexperiments/experiment.py [hunk 1]
```diff
@@ -24,6 +24,22 @@ class Experiment(_base_types.Record):
     add a new trial to an Experiment by calling :meth:`~smexperiments.experiment.Experiment.create_trial`.
     To remove a Trial from an experiment, delete the trial.
 
+    Examples:
+        .. code-block:: python
+
+            from smexperiments import experiment
+
+            my_experiment = experiment.Experiment.create(experiment_name='AutoML')
+            my_trail = my_experiment.create_trial(trial_name='random-forest')
```
Comment (danabens) [inline src/smexperiments/experiment.py]: "trial"

File: src/smexperiments/experiment.py [hunk 2]
```diff
@@ -24,6 +24,22 @@ class Experiment(_base_types.Record):
...
+
+            for exp in experiment.Experiment.list():
+                print(exp)
+            for trial in my_experiment.list_trials():
+                print(trial)
+
+            my_trail.delete()
```
Comment (danabens) [inline src/smexperiments/experiment.py]: ditto

File: src/smexperiments/experiment.py (hunk 1 above)
Comment (l2yao) [reply src/smexperiments/experiment.py]: nice catch, fixed

File: src/smexperiments/experiment.py (hunk 2 above)
Comment (l2yao) [reply src/smexperiments/experiment.py]: fixed

**Pull request merged**